AuditService.Web/Controllers/AuditController.cs

```diff
@@ -15,14 +15,16 @@ public AuditController(IAuditService auditService)
     }
 
     [HttpGet]
-    public async Task<IActionResult> GetAuditChanges([FromQuery] Guid userId, [FromQuery] DateTime at)
+    public async Task<IActionResult> GetAuditChanges([FromQuery] Guid userId, [FromQuery] DateTime at, [FromServices] IMonolithClient monolithClient)
     {
         var list = await _auditService.GetAuditChangesAsync(userId, at);
+        var userLogins = await monolithClient.GetUserLoginAsync(userId);
```

Comment:
I think you should enrich changes inside the audit service. In other case your endpoint grows